lib/Backend/LinearScan.cpp

```diff
@@ -5050,13 +5035,7 @@ IR::Instr* LinearScan::GeneratorBailIn::GenerateBailIn(IR::GeneratorBailInInstr*
         bailInInstr->capturedValues
     );
 
-    this->InsertRestoreSymbols(
-        *bailOutInfo->byteCodeUpwardExposedUsed,
-        bailInInstr->upwardExposedUses,
-        bailInInstr->capturedValues,
-        insertionPoint
-    );
-    Assert(!this->func->IsStackArgsEnabled());
```

Comment:
This Assert had gone adrift from the comment that explained it - therefore moved upwards.